src/accordion/accordion.ts

```diff
@@ -113,6 +118,16 @@ export class NgbAccordion implements AfterContentChecked {
   @ContentChildren(NgbPanel) panels: QueryList<NgbPanel>;
 
   /**
+   * A map that stores each panel state
```

Comment:
Thanks for moving them in place!
But could you move them a little higher, before the `panels`?

https://github.com/ng-bootstrap/ng-bootstrap/wiki/Code-conventions#component-internal-organization